eip-0034.md

```diff
@@ -0,0 +1,121 @@
+# NFT Collection Standard V1
+
+* Author: anon_real
+* Status: Proposed
+* Created: 10-May-2022
+* License: CC0
+* Forking: not needed 
+
+## Motivation 
+NFT collections play a central role in the nowadays NFT ecosystem on all blockchains. It gives the ability to artists to organize their artworks and present them in a more meaningful way. It also can facilitate the use of other NFT features such as traits. As we'll present in this proposal, it can also enable more advanced features such as artist collaborations. These advanced features can get arbitrary complex with the use of smart contracts.
+
+## Design
+We consider collection NFTs as tokens! The issued collection token will contain all the collection information.
+
+An NFT will be considered as part of an already created collection if it follows the [artwork standard]() and the collection NFT is present in the first input of the issuance transaction which is known as the royalty box.
+
+There is no limitation on the number of issued tokens for a collection. One possible interpretation of such a feature is that artists can come together and create a collection. Each one of them will have one collection token which gives the right to them to add NFTs to that collection. We believe that with the use of smart contracts, more advanced usage of this flexibility will be discovered in the future.
+
+## NFT Collection Standard
+One of the main goals of this proposal is for the standard to be as future-proof as possible. To achieve that, some of the features we introduce here may not contain an extensive explanation of how to use them. This enables NFT developers to participate in the EIP and also plan for their tools. However, it comes with the downside that this EIP will be prone to backward-compatible extensions in the future.
+
+### Collection standard version
+Collection standard version must be present in `R4` as `Int`. This current standard version is `1`. This will enable future radical changes depending on the state of the NFT ecosystem.
+
+### Collection Info
+Collection info must be present in `R5` of the issuance box as `Coll[Byte]`.
+The following JSON contains the structure of collection info.
+```json
+{
+    "name": "NAME_OF_THE_COLLECTION",
+    "description": "DESCRIPTION_OF_THE_COLLECTION",
+    "logo": "URL_TO_COLLECTION_LOGO",
+    "featured_image": "URL_TO_COLLECTION_FEATURED_IMAGE",
+    "banner_image": "URL_TO_COLLECTION_BANNER_IMAGE",
+    "category": "COLLECTION_CATEGORY",
+    "explicit_content": true or false
+}
+```
```

Comment:
Not sure if should be a collection parameter or NFT's, but copyright rights can be interesting to have. 
NFT artist should define what the owner is allowed to use the NFT for: commercial use, modifications,